nixos/doc/manual/release-notes/rl-1404.xml

```diff
@@ -13,7 +13,7 @@ the following highlights:
 <itemizedlist>
 
   <listitem><para>Installation on UEFI systems is now supported.  See
-  <xref linkend="sec-uefi-installation"/> for
+  <xref linkend="sec-installation"/> for
```

Comment:
I don't see a match in these changes for where the the docbook section for sec-uefi-installation is renamed.

Reply:
The docbook section for sec-uefi-installation is removed, as the instructions were folded into `installing.xml`

Reply:
Oops, missed that in my commit, just pushed ammended to do the actual removal :)